wiki/Jobs.md

```diff
@@ -85,3 +85,50 @@ First create a project. A project is a collection of jobs. Then click the add ic
     ]
 }
 ```
+
+### Example with pull request in checkout.
+
+Note: the checkout Will fail if there are merge conflicts. For success the output will show the specific sha for the
+pulled branch.
+
+```json
+{
+    "name": "Example Job",
+    "platform": "android",
+    "cron_time": "0 0 * * *",
+    "device_tag": "device_type_1",
+    "notify": "example@yahoo.com",
+    "checkout": {
+        "type": "javascript",
+        "task": "git-checkout",
+        "checkout_url": "git@github.com:yahoo/SomeAndroidProject.git",
+        "branch": "master",
+        "pull": ["origin", "branch-to-pull"]
+    },
```

Comment:
In some cases we could do less. As you pointed out in some cases the pull reduces to a merge. But the current technique opens up the ability pull from somewhere else as well. Perhaps a user  wants to pull from some open-source project that is external to the current repo.